test/integration/filters/stream_info_to_headers_filter.cc

```diff
@@ -24,8 +24,31 @@ class StreamInfoToHeadersFilter : public Http::PassThroughFilter {
       headers.addCopy(Http::LowerCaseString("alpn"),
                       decoder_callbacks_->streamInfo().upstreamSslConnection()->alpn());
     }
+
     return Http::FilterHeadersStatus::Continue;
   }
+  Http::FilterTrailersStatus encodeTrailers(Http::ResponseTrailerMap& trailers) override {
+    StreamInfo::UpstreamTiming& upstream_timing = decoder_callbacks_->streamInfo().upstreamTiming();
+    // Upstream metrics aren't available until the response is complete.
```

Comment:
Can we add this into the comment on the `StreamInfo::upstreamTiming()` API, please?